### PR TITLE
v4 authentication methods and examples

### PR DESCRIPTION
**Pull request by lepunk**
## Context
This SDK is linked from our docs and it needs to be updated to V4

## Known issues
The merchant API docs are lying about the /verticals endpoint. This PR handles that endpoint well but either the docs or the API needs to be updated

### CODE REVIEW

**Review by mikone117 — APPROVED** [review]
LGTM 🥇

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1 @@
 requests==2.20.0
-click==6.7
```
Comment (pcalessio) [inline requirements.txt:2]: 😞

**Review by pcalessio — APPROVED** [review]
👍

**Comment by neenu-ram-skim** [comment]
Looks good to me 👍

**Pull request merged**